src/codegen.cpp

```diff
@@ -4376,6 +4385,23 @@ static void emit_varinfo_assign(jl_codectx_t &ctx, jl_varinfo_t &vi, jl_cgval_t
     }
 }
 
+static void emit_binding_store(jl_codectx_t &ctx, jl_binding_t *bnd, Value *bp, jl_value_t *r, ssize_t ssaval, AtomicOrdering Order)
+{
+    assert(bnd);
+    Value *rval = boxed(ctx, emit_expr(ctx, r, ssaval));
+    if (!bnd->constp && bnd->ty && jl_isa(r, bnd->ty)) {
+        StoreInst *v = ctx.builder.CreateAlignedStore(rval, bp, Align(sizeof(void*)));
+        v->setOrdering(Order);
+        tbaa_decorate(ctx.tbaa().tbaa_binding, v);
+        emit_write_barrier_binding(ctx, bp, rval);
```

Comment:
```suggestion
        emit_write_barrier_binding(ctx, literal_pointer_val(ctx, bnd), rval);
```

since `bp == julia_binding_gv(ctx, literal_pointer_val(ctx, bnd))`